cwms-data-api/src/test/java/cwms/cda/data/dto/location/kind/CharacteristicRefTest.java

```diff
@@ -0,0 +1,41 @@
+package cwms.cda.data.dto.location.kind;
+
+import cwms.cda.formatters.ContentType;
+import cwms.cda.formatters.Formats;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class CharacteristicRefTest
+{
+	private static final String SPK = "SPK";
+	private static final String CHARACTERISTIC = "Characteristic";
```

Comment:
Please change the value to something that references a real characteristic for those not as familiar with the various concepts.

Reply:
I couldn't find any characteristics in our SWT databases, but I found documentation for REGI that included the characteristic of 'ogee weir depth'.  I hope that's more acceptable.